src/Widget/Attribute/FieldAttributes.php

```diff
@@ -18,53 +17,123 @@
 trait FieldAttributes
```

Comment:
There are duplicate methods such as `containerClass` and `withContainerClass`.

Reply:
done.